.github/resources/.actions/cli-runner/action.yml

```
@@ -0,0 +1,130 @@
+name: Slack CLI Installation and Command Runner 
+description: Download and cache the Slack CLI and run the input command 
+
+inputs:
+  command:
+    description: "Slack CLI command to run"
+    type: string
+    default: ""
+    required: true
+  verbose:
+    description: "Verbose flag"
+    type: boolean
+    default: false
+    required: false
+  cli_version:
+    description: "CLI Version"
+    type: string
+    default: "latest"
+    required: false
+  app_id: 
+    description: "App ID"
+    type: string
+    default: ""
+    required: false
+
+outputs:
+  success:
+    description: "Whether the command ran successfully"
+    value: ${{ steps.run-slack-cli-command.outputs.success }}
```

```suggestion
  ok:
    description: "Whether the command ran successfully"
    value: ${{ steps.run-slack-cli-command.outputs.ok }}
```

📺 suggestion: To match a [similar](https://github.com/slackapi/slack-github-action/blob/a560f02f1e4a8dee2a30c9f08cf9f8a1d04c3f11/action.yml#L44) value of other techniques-